src/passes/MemoryPacking.cpp

```diff
@@ -87,6 +87,10 @@ Expression* makeShiftedMemorySize(Builder& builder) {
 struct MemoryPacking : public Pass {
   size_t dropStateGlobalCount = 0;
 
+  // FIXME: Chrome has a bug decoding section indices that prevents it from
+  // using more than 63. Just use MaxDataSegments once this is fixed.
+  uint32_t MAX_SEGMENTS;
```

Comment:
Why all caps?

Reply:
It's not actually a constant, but it does fill the role of a constant, so I thought it would be appropriate to name it like a constant. I can change this if you'd like.

Reply:
No worries.

Reply:
Might want to reference this bug so you know when this can be removed: https://bugs.chromium.org/p/v8/issues/detail?id=10151

Reply:
+1 to renaming it, +1 to linking to the bug with a comment like "temporary workaround" etc.